reverse_engineering/api.js

```diff
@@ -138,7 +138,7 @@ module.exports = {
 			]);
 			callback(
 				null,
-				mergeCollectionsWithViews(jsonSchemas),
+				mergeCollectionsWithViews({ jsonSchemas, _ }),
```

Comment:
I think with the new build system we can just install and use lodash as a dependency and do not import it from the app.